src/db/getOneRecord.js

```diff
@@ -0,0 +1,14 @@
+import AWS from 'aws-sdk';
+import { config } from '../config/dynamoConfig';
+
+export const getOneRecord = async (pk, name) => {
+    const client = new AWS.DynamoDB.DocumentClient();
+    const params = {
+        TableName: config.tableName,
+        Key: {
+            id: pk,
```

Comment:
you could make this a bit cleaner by renaming `pk` to `id`
then this would become `Key: { id, name }`